cyhy_report/customer/report.mustache

```diff
@@ -755,6 +759,46 @@ As you review the report, you may have additional questions. Check out the answe
   \end{tikzpicture}
 \restoregeometry
 
+<<#owner_is_federal_executive>>
+\section{Binding Operational Directive 22-01 --- Reducing the Significant Risk of Known Exploited Vulnerabilities}<</owner_is_federal_executive>><<^owner_is_federal_executive>>\section{Reducing the Significant Risk of Known Exploited Vulnerabilities}<</owner_is_federal_executive>>
+\label{sec:known-exploited-vulnerabilities}
+
+Malicious cyber campaigns frequently use \glspl{KEV} to threaten the public sector, the private sector, and ultimately the security and privacy of individual citizens. Therefore it is essential to quickly remediate \glspl{KEV} to protect <<#owner_is_federal_executive>>federal<</owner_is_federal_executive>><<^owner_is_federal_executive>>your<</owner_is_federal_executive>> information systems and reduce cyber incidents<<^owner_is_federal_executive>> affecting critical infrastructure<</owner_is_federal_executive>>.
+
+<<#owner_is_federal_executive>>\gls{CISA} issued \href{https://www.cisa.gov/binding-operational-directive-22-01}{Binding Operational Directive (BOD) 22-01: Reducing the Significant Risk of Known Exploited Vulnerabilties} to evolve the federal government's approach to \gls{vulnerability} management and keep pace with threat activity.  The directive establishes a \href{https://www.cisa.gov/known-exploited-vulnerabilities}{CISA managed catalog} of \glspl{known exploited vulnerability} and requires federal civilian agencies to identify and remediate these vulnerabilities found on your information systems within two weeks.<</owner_is_federal_executive>><<^owner_is_federal_executive>>\gls{CISA} established a \href{https://www.cisa.gov/known-exploited-vulnerabilities}{CISA managed catalog} of \glspl{known exploited vulnerability} and requires federal civilian agencies to identify and remediate these \glspl{vulnerability} found on their information systems within two weeks \textit{- and emphatically recommends that all information system owners and operators do the same.}<</owner_is_federal_executive>>
```

Comment:
I'd prefer if we word-wrapped paragraphs like this.  As it stands, it's a real pain to review in GH.

I know we're inconsistent with word-wrapping on our legacy code, but I tend to do it when I have to go in any monkey with the LaTeX source.

Reply:
In `emacs` the command is `fill-paragraph`, bound by default to `Alt-q`, if that helps.

Reply:
> In `emacs` the command is `fill-paragraph`, bound by default to `Alt-q`, if that helps.

You know that it doesn't! 🤣 

However I did install the VS Code ["Rewrap" extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) (which pays tribute to `emacs` by also binding to `Alt-q` by default), so enjoy the pleasantly-wrapped goodness in https://github.com/cisagov/cyhy-reports/pull/71/commits/9597bf0dc06e7ad0e7fb7c51474698836cc71de8.

Reply:
VS Code?!?  What heresy is this?!?

Reply:
I'm trying it out and so far, I don't hate it.  Only myself for not hating it.